src/main/java/ledger/user_interface/ui_controllers/MainPageController.java

```diff
@@ -150,7 +150,8 @@ private void createAccountPopup() {
      */
     private void deleteAccount() {
         if (chooseAccount.getSelectedAccount() == null) {
-            setupErrorPopup("Cannot delete the All Accounts aggregation", new Exception());
+            setupErrorPopup("Cannot delete the All Accounts aggregation. Please select an account " +
+                    "to delete.", new Exception());
```

Comment:
My concern was more just over that it was of type Exception. If want the stack trace should probably make something that extends LedgerException (UserInputException?) If don't need it could also have the method handle null.